corehq/tests/noseplugins/timing.py

```diff
@@ -114,3 +125,32 @@ def end_event(name, context):
     event. Requires the `TimingPlugin` must to be enabled.
     """
     PLUGIN_INSTANCE.end_event(name, context)
+
+
+def _get_expected_slow_seconds(event, test_context):
+    attr_name = {
+        "before": "slow_before",
```

Comment:
Doesn't look like this is used anywhere (or [even documented](https://github.com/dimagi/commcare-hq/pull/23874/commits/06577972f57141e71af205e78daa62562723f334)). Do you think it would make sense to remove it until we need it? If not, what do you think of documenting it, clearly distinguishing the difference between `slow_setup` and `slow_before`?

Reply:
https://github.com/dimagi/commcare-hq/pull/23874/commits/6bbb8e6172c77ffa0220b05e31fb545bfe3b6451